### ISSUE TITLE
charts throw JS error when clicking 'render visualization' button

### ISSUE THREAD

**Issue by martenson** [galaxyproject/galaxy#4851]
This can be seen on Test.

```
utils.js:380 Uncaught ReferenceError: datetime is not defined
    at Object.time (utils.js:380)
    at n._drawChart (editor.js:144)
    at Object.onclick (editor.js:22)
    at HTMLDivElement.<anonymous> (ui-buttons.js:367)
    at HTMLDivElement.dispatch (jquery.js?v=1508772179:1)
    at HTMLDivElement.v.handle (jquery.js?v=1508772179:1)
```

**Comment by dannon** [comment]
@martenson @guerler Mentioned something weird with charts, too.  I'll take a look now, thanks for the traceback.  (edit: I see what's broken here, should have a fix in a few)

**Comment by dannon** [comment]
Fixed in #4852.  Thanks again for the traceback.
